src/utils/feeBreakdown.ts

```diff
@@ -162,3 +189,20 @@ export const getFeeBreakdown = (passportData: unknown): FeeBreakdown => {
 
   return toFeeBreakdown(parsedPassport.data);
 };
+
+/**
+ * Parse passport against schema then transform to a ReductionOrExemption object
+ */
+export const getCalculatedReductionOrExemption = (
+  passportData: unknown,
+): ReductionOrExemption => {
+  const parsedPassport = schema.safeParse(passportData);
+
+  if (!parsedPassport.success) {
+    throw Error("Failed to parse reduction or exemption fee breakdown data", {
+      cause: parsedPassport.error.flatten(),
+    });
+  }
+
+  return calculateReductionOrExemptionAmounts(parsedPassport.data);
+};
```

Comment:
Previously, we just needed the concept of a "calculated" reduction or exemption (transformed boolean to number) at the point of the fee breakdown (aka `Pay`), but now with the addition of VAT we additionally need to be able to reference this same logic in `SetFee` to ensure that all VAT is correctly handled. 

Hence we now export `getCalculatedReductionOrExemption` _and_ `getFeeBreakdown`.